.github/workflows/e2e-tests.yml

```diff
@@ -4,9 +4,9 @@ on:
     branches:
       - master
   pull_request:
-    paths:
-      - '**'
-      - '!doc/**'
+    paths-ignore:
+    - 'doc/**'
+    - '**/*.md'
```

Comment:
`./doc/install/local-values.yaml` used to be used in manifest generation, not sure if that's still true though.

Reply:
Updated to use `'doc/**/*.md` after looking at the examples outlined in https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-file-paths.